composer.json

```diff
@@ -20,7 +20,10 @@
     "repositories": [
         {
             "type": "git",
-            "url": "git@github.com:rdlowrey/aerys"
+            "url": "git@github.com:amphp/aerys.git"
         }
-    ]
+    ],
+    "config": {
+      "bin-dir": "bin"
+    }
```

Comment:
I don't see a real reason to have that copy actually, it just saves a single path segment.